test/results/oneway_model.json

```diff
@@ -0,0 +1,10 @@
+{
+  "OneWayRequest": {
+    "name": "OneWayRequest",
+    "properties": {
+      "OneWayField": {
+        "type": "string"
+      }
+    }
+  }
+}
```

Comment:
Missing terminating newline. You may need to configure your editor to do this properly.